compiler-rt/test/sanitizer_common/TestCases/sanitizer_coverage_trace_pc_guard.cpp

```diff
@@ -2,6 +2,8 @@
 
 // REQUIRES: has_sancovcc
 // UNSUPPORTED: ubsan,i386-darwin,target={{(powerpc64|s390x|thumb).*}}
+// This test is failing for lsan on darwin on x86_64.
+// UNSUPPORTED: x86_64h && lsan
```

Comment:
Would we want to constrain disablement to Apple platforms? (` && darwin`)

Reply:
I believe the lit feature x86_64h is only added on Apple platforms. So I don't think the logic would actually add anything, though it would make it more clear.